src/components/Container/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { PropsWithChildren } from 'react';
+import styled, { CSSProperties } from 'styled-components';
+
+interface ContainerProps extends PropsWithChildren {
+  maxWidth?: number;
+  minWidth?: number;
+  backgroundColor: CSSProperties['backgroundColor'];
+}
+
+const ContainerComponent = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+`;
+
+const ContainerContentComponent = styled.div<ContainerProps>`
+  min-width: ${({ maxWidth, minWidth }) => {
+    const maxWidthParams = !maxWidth ? '100%' : `${maxWidth}px`;
+    return `${!minWidth ? maxWidthParams : `${minWidth}px`}`;
+  }};
```

Comment:
```suggestion
  min-width: ${({ maxWidth, minWidth }) => {
    const maxWidthParams = `${maxWidth}px` || '100%';
    return minWidth || maxWidthParams;
  }};
```
개인적으론 부정문이 들어간 삼항연산자는 이해가 어려운 것 같아서,,
or를 쓰면 어떨까요?

Reply:
```js
    const maxWidthParams = `${maxWidth}px` || '100%';
```
maxWidth이 없어도 첫번째 식은 px로 평가되므로 참이 될 것 같습니다..!
```js
    return minWidth || maxWidthParams;
```
이 친구도 minWidth자체에 px를 붙여주어야해서, 어쩔수 없었습니다 흑흑 😅